backward.hpp

```diff
@@ -1590,7 +1590,7 @@ class SourceFile {
 		swap(from); return *this;
 	}
 #else
-	explicit SourceFile(const SourceFile& from) {
```

Comment:
This is not a good idea. It will allow implicit move semantic. Which is going to corrupt the from argument silently. Forcing it explicit ensures that you somewhat know what you are doing. A solution is maybe to make both the move constructor and the poor's man move constructor private. And then write `move` methods wrapping them. Which is by definition explicit to call.

`move(const SourceFiles* from)` on <C++11

and for >=C++11

`move(SourceFile&& from)`

Reply:
I am taking the suggestion.